recipes/bob.ap/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.ap
+  version: {{ version }}
+
+source:
+  fn: bob.ap-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.ap/bob.ap-{{ version }}.zip
+  md5: 49f89a84c3c2cb20aa0ffa49d39415a7
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.sp
+    - scipy
```

Comment:
Usually `bob` packages link against `bob.blitz` and `bob.math` and not against `numpy` directly. I don't see any direct usage of `numpy` here:(https://github.com/bioidiap/bob.ap/search?l=cpp&q=numpy&utf8=%E2%9C%93)